pkg/kubectl/cmd/get/get.go

```diff
@@ -581,7 +581,12 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	w.Flush()
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
-		noResourcesFoundFormat := fmt.Sprintf("No resources found in %s namespace.", o.Namespace)
+		noResourcesFoundFormat := ""
+		if !o.AllNamespaces {
```

Comment:
Sorry about this but, do you need `noResourcesFoundFormat`, you could call `Fprintln` directly